Javascript/talismans.js

```diff
@@ -1,207 +1,245 @@
+  document.getElementById("talismanlevelup").style.display = "none"
+  document.getElementById("talismanEffect").style.display = "block"
+  document.getElementById("talismanrespec").style.display = "none"
+  let a = document.getElementById("talismanSummary")
+  let b = document.getElementById("talismanBonus")
+  let c = document.getElementById("talismanRune1Effect")
+  let d = document.getElementById("talismanRune2Effect")
+  let e = document.getElementById("talismanRune3Effect")
+  let f = document.getElementById("talismanRune4Effect")
+  let g = document.getElementById("talismanRune5Effect")
+  let h = document.getElementById("talismanMythicEffect")
+
+  let index = player.talismanRarity[i]
+  let modifiers = [null, "+", "+", "+", "+", "+"]
+  let num = talismanPositiveModifier[index];
+  let talismanRarityMult = [null, num, num, num, num, num]
+
+  for (var j = 1; j <= 5; j++) {
+    if (player["talisman" + ord[i]][j] < 0) {
+      modifiers[j] = "-";
+      talismanRarityMult[j] = talismanNegativeModifier[index]
     }
-    if(player.talismanRarity[i] !== 6){h.textContent = "Get Max Enhance for a Mythical bonus effect!"}
-}
-
-function showTalismanPrices(i){
-    document.getElementById("talismanEffect").style.display = "none"
-    document.getElementById("talismanlevelup").style.display = "block"
-    document.getElementById("talismanrespec").style.display = "none"
-    let a = document.getElementById("talismanShardCost")
-    let b = document.getElementById("talismanCommonFragmentCost")
-    let c = document.getElementById("talismanUncommonFragmentCost")
-    let d = document.getElementById("talismanRareFragmentCost")
-    let e = document.getElementById("talismanEpicFragmentCost")
-    let f = document.getElementById("talismanLegendaryFragmentCost")
-    let g = document.getElementById("talismanMythicalFragmentCost")
-
-    document.getElementById("talismanLevelUpSummary").textContent = "-=-=- Resources Required to Level Up -=-=-"
-    document.getElementById("talismanLevelUpSummary").style.color = "silver"
-
-    let m = talismanLevelCostMultiplier[i]
-    a.textContent = format(m * Math.max(0, Math.floor(1 + 1/8 * Math.pow(player.talismanLevels[i],3))));
-    b.textContent = format(m * Math.max(0, Math.floor(1 + 1/32 * Math.pow(player.talismanLevels[i] - 30,3))));
-    c.textContent = format(m * Math.max(0, Math.floor(1 + 1/384 * Math.pow(player.talismanLevels[i] - 60,3))));
-    d.textContent = format(m * Math.max(0, Math.floor(1 + 1/250 * Math.pow(player.talismanLevels[i] - 90,3))));
-    e.textContent = format(m * Math.max(0, Math.floor(1 + 1/125 * Math.pow(player.talismanLevels[i] - 120,3))));
-    f.textContent = format(m * Math.max(0, Math.floor(1 + 1/64 * Math.pow(player.talismanLevels[i] - 150,3))));
-    g.textContent = format(m * Math.max(0, Math.floor(1 + 1/128 * Math.pow(player.talismanLevels[i] - 150,3))));
+  }
+
+  switch (i) {
+    case 1:
+      a.textContent = "=-=-=-= Exemption Talisman Effects =-=-=-=";

```

Comment:
Again, here use the fancy ES6 method. For example:
```js
g.textContent = `Bonus SI Rune Levels: ${format(talisman1Effect[5], 2, true)}`
```

Reply:
Done